package.json

```diff
@@ -8,9 +8,9 @@
     "build": "microbundle -f iife",
     "dev": "run-all \"serve\" \" microbundle watch -f iife \"",
     "deploy": "npm run build",
-    "test-setup": "node test/setup/checkout-wpt.mjs",
     "test:wpt": "npm run test-setup && cd test && cd wpt && (python wpt run --headless -y --log-wptreport ../report/data.json --log-wptscreenshot=../report/screenshots.txt --log-html=../report/index.html --inject-script ../../dist/scroll-timeline.js firefox scroll-animations || true)",
     "test:simple": "npm run test-setup && cd test && cd wpt && python wpt serve --inject-script ../../dist/scroll-timeline.js",
+    "test-setup": "node test/setup/checkout-wpt.mjs && npm run build",
```

Comment:
I'm surprised you regularly use test-setup. You want to pull the latest from wpt as part of your build and test flow?

If we make `npm run build` part of the `test-setup` can you remove the extra call to `npm run build` from gh-pages.yml?

Reply:
I don’t directly run `npm run test-setup`, but do want a fresh build when running `test:wpt` or `test:simple` which rely on `test-setup`. That’s why I added it here.

Reply:
Ah, makes sense.

Reply:
Done.